projects/packages/publicize/src/class-wpcom-rest-api-v2-endpoint-list-publicize-connections.php

```diff
@@ -162,7 +225,9 @@ protected function get_connections() {
 	public function get_items( $request ) {
 		$items = array();
 
-		foreach ( $this->get_connections() as $item ) {
+		$connections = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? $this->get_connections() : $this->get_connections_from_wpcom();
```

Comment:
We need to have this logic in a static method like `get_connections` in #40607 to make it usable directly in script data.

Reply:
Also, connection test results will now need to be a separate endpoint because this one doesn't incorporate it and we will need to have the `status` field in that.